upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -378,13 +378,14 @@ func (c *ApplyClusterCmd) Run() error {
 				//"dnsZone": &awstasks.DNSZone{},
 			})
 
+			// TODO: We need to figure out how to safely skip this check when users really don't want an SSH key
```

Comment:
We probably need an override value in the API.  `doNotConfigureSSHKey` or something like that.